contents/docs/tutorials/retention.md

```diff
@@ -0,0 +1,174 @@
+
+[**Stickiness**](/docs/features/trends/#trend-segmentation-by-stickiness)
+
+Stickiness is also a metric of repeated usage, but it disregards the absolute time period in favor of a relative metric. In practical terms, while retention will give you an answer for how many users visited your page on October 22nd and then visited again on October 23rd, stickiness doesn't care about the dates, rather offering an answer for how many users came back to your site on day 2 after visiting on day 1 (irrespective of what the exact dates were). Tracking stickiness in PostHog is done in the 'Trends' tab in 'Insights'.
+
+![Retention in Insights Image](../../images/tutorials/retention/insights.png)
+
+This tutorial will focus on 'retention' specifically, but the majority of concepts is also applicable to stickiness views.
+
+#### Configuring Retention Tables for your Metrics
+
+To start calculating your retention, head over to the 'Retention' tab in 'Insights'.
+
+This will show you a retention table for the `$pageview` event, like so:
+
+![Retention Table Image](../../images/tutorials/retention/table.png)
+
+If this table is a bit confusing to you, check out our dedicated [retention](/docs/features/retention) page for a full walkthrough of how it works.
+
+Here, you have a few settings you can tweak to meet your desired use-case:
+
+##### Track retention on the events that matter to you
+
+The first thing you can do is select any action or event to track retention on. This can be:
+
+- A website visit: Shows you if your users are coming back repeatedly
+- A button click: Can be used to track usage of important features you want users to use often, such as comment on a post 
+- A custom event: Anything you want users to do regularly (see [custom events](/docs/integrations/js-integration/#sending-events))
+
+Remember that here you are tracking how often users come and do the same thing again over a period of time. To track subsequent actions or events, you should use [funnels](/docs/features/funnels).
+
+##### Setting relevant filters
+
+Even if you don't have complex metrics in place, the filter functionality should not be overlooked. It can be used to:
+
+**Segregate events by URL**
+
+If you have multiple domains being tracked in PostHog at once, you might want to look at their retention metrics separately. This can be done through filtering by 'Current URL'. In our case, for example, to look at retention for [app.posthog.com](https://app.posthog.com) (our product) separately, we set a filter of `Current URL contains app.posthog.com`.
+
+In addition, you could also look at retention for specific pages on your website, such as for your blog or documentation.
+
+**Determining product-market fit**
+
+Informing product direction for better product-market fit is one of the main use-cases for retention.
```

Comment:
I'd add a tiny bit more context here as I don't know if this will be self-explanatory for everyone.

For example, something like:

```
Traditionally, you are supposed to focus on your ability to sell a product to work out if you have product market fit. However, we believe it is *just* as important that users are coming back to your product repeatedly. If you can get both of those things right, you know you are in a position to scale your business. Retention tells you if people are coming back.

It's also key to recognize that you may have stronger product market fit with a subset of your users - for example, those in a certain group or cohort. If you've time, check out [Superhuman's explanation for this](https://firstround.com/review/how-superhuman-built-an-engine-to-find-product-market-fit/) - they used surveys, but we believe that whilst both have value, what people do is more important than what they say.

```
... (to understand what groups you retain better)